multiqc/modules/longranger/longranger.py

```diff
@@ -0,0 +1,330 @@
+#!/usr/bin/env python
+
+""" MultiQC module to parse output from Longranger """
+
+from __future__ import print_function
+from collections import OrderedDict
+import logging
+import re
+import os
+import csv
+
+from multiqc import config
+from multiqc.modules.base_module import BaseMultiqcModule
+from multiqc.plots import table
+
+# Initialise the logger
+log = logging.getLogger(__name__)
+
+class MultiqcModule(BaseMultiqcModule):
+    """ Longranger module """
+
+    def __init__(self):
+
+        # Initialise the parent object
+        super(MultiqcModule, self).__init__(name='Longranger', anchor='longranger',
+        href="https://www.10xgenomics.com/",
+        info="A set of analysis pipelines that perform sample demultiplexing, "
+        "barcode processing, alignment, quality control, variant calling, phasing, "
+        "and structural variant calling.")
+
+        self.headers = OrderedDict()
+        self.headers['longranger_version'] = {
+                'scale': False,
+                'title': 'Version',
+                'description': 'The version of the Long Ranger software used to generate the results.',
+                'hidden': True
+        }
+        self.headers['instrument_ids'] = {
+                'scale': False,
+                'title': 'Instrument ID',
+                'description': 'The list of instrument IDs used to generate the input reads.',
+                'hidden': True
+        }
+        self.headers['gems_detected'] = {
+                'scale': 'RdYlGn',
+                'title': '# gems',
+                'modify': lambda x: float(x) / 1000000.0,
+                'suffix': 'M',
```

Comment:
I think that the standard way of doing millions etc. is to add the `M` in the title (suffix is more for `%` etc). So for this one: `'title': 'M Gems'`. Also please add `(millions)` to the end of the description text.

Same goes for the other columns.